src/Language/PureScript/Errors.hs

```diff
@@ -1060,6 +1061,12 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
             , indent $ paras $ map (line . markCode . showQualified showOp) (NEL.toList ops)
             , line "Use parentheses to resolve this ambiguity."
             ]
+    renderSimpleErrorMessage NeedToBuildDocs =
+      paras
+        [ line "Some of the input modules are missing up-to-date documentation data in the compiler output directory. Please compile all of the relevant modules using `purs compile' with the `docs' codegen target."
+        , line "For example:"
+        , indent $ line "$ purs compile --codegen js,docs SOURCE_FILES"
```

Comment:
Is the `js` option necessary? I don't think it is, and I'm sure the example is meant to indicate "hey, you can compile docs with normal JS anytime you want", but it makes it seem likes it's necessary.

Reply:
You're right, it isn't necessary. Do you think we should just write `--codegen docs` here then? I don't have any strong feelings either way.

Reply:
I don't think it's necessary. The help for `purs compile` indicates that you can provide multiple codegen targets.